imports/plugins/core/router/client/startup.js

```diff
@@ -14,22 +14,21 @@ Meteor.startup(function () {
   // Note: Although these are subscribed to by the subscription manager in "/modules/client/core/subscriptions",
   // using the subscriptions manager sometimes causes issues when signing in/out where you may seee a grey screen
   // or missing shop data throughout the app.
-  // TODO: Revisit subscriptions manager usage and waiting for shops to exist client side before rendering.
-  const primaryShopSub = Meteor.subscribe("PrimaryShop");
+  const shopSub = Meteor.subscribe("Shop");
```

Comment:
looking closer, I don't believe this change is necessary (although, from what I can tell, not harmful either)

Reply:
If this change isn't necessary, let's remove it. Not sure what it might affect in 3rd party plugins or other shops since it's a managed subscription.